src/main/java/org/genomicsdb/importer/GenomicsDBImporter.java

```diff
@@ -537,16 +556,22 @@ private void iterateOverSamplesInBatches(final int sampleCount, final int update
         final boolean failIfUpdating = this.config.getImportConfiguration().getFailIfUpdating();
         final int totalBatchCount = (sampleCount + updatedBatchSize - 1)/updatedBatchSize; //ceil
         BatchCompletionCallbackFunctionArgument callbackFunctionArgument = new BatchCompletionCallbackFunctionArgument(0, totalBatchCount);
-        for (int i = 0, batchCount = 1; i < sampleCount; i += updatedBatchSize, ++batchCount) {
+        int origLbRowIdx = (this.config.getImportConfiguration().hasLbCallsetRowIdx())
```

Comment:
In `origLbRowIdx`, what does the `Lb` stand for?

Reply:
Yes, it is lower bound. The index that we start importing from in the callset